In [1]:
a <-read.csv("donation.csv", header=TRUE, sep=",")

Question 1:

In [2]:
# Calculate for the total donation for each alumni
a$TGiving=a$FY04Giving+a$FY03Giving+a$FY02Giving+a$FY01Giving+a$FY00Giving

In [3]:
# Calcualte the mean for donation per person
mean=mean(a$TGiving)

In [4]:
print(mean)

[1] 980.0436


In [5]:
#the number of total alumus
numa=nrow(a) 

In [6]:
print(numa)

[1] 1230


*since we are using sample standard deviation to estimate population standard deviation, even though the sample size is decent, we still use T distribution for more accurate result.

In [7]:
# with 95% confidence, we may know a/2=(1-95%)/2=2.5%, so one tail of the distribution should be 2.5%. 
t=qt(0.975,numa-1) 

In [8]:
# below we calculate the 95% confident interval
CIta=mean+c(-1,1)*t*sd(a$TGiving)/sqrt(numa)

In [9]:
print(CIta)

[1]  606.8797 1353.2074


Question 1: We are 95% confident that the true mean donation is between 606.8797 and 1353.2074 USD, although the true mean may or may not be in this interval, 95% of the intervals formed in this manner from all samples will contain the true mean. 

Question 2:

In [10]:
# calculate the number of alumus who donate at least 1000usd
b=a[a$TGiving>=1000,]

In [11]:
numb=nrow(b) 

In [12]:
# the poportion of the alumus donated at least 1000usd
p=numb/numa

In [13]:
z=qnorm(0.975)

In [14]:
# calculate the confident interval for the above poportion
CIp=p+c(-1,1)*z*sqrt(p*(1-p)/1230)

In [15]:
print(CIp)

[1] 0.09304368 0.12809453


Question 2: We are 95% confident that the true percentage of alumni who donate equal or more than 1000 USD in the population is between 9.30% and 12.81%. 

Question 3:

In [16]:
# assign sampling error 100 to "e"
e=100

In [17]:
# assign the required sample size to N1
N1=z^2*(sd(a$TGiving))^2/e^2

In [18]:
print(N1)

[1] 17094.19


In [19]:
#Original sample size
print(numa)

[1] 1230


In [20]:
#Let's see the original Sampling error
print(c(-1,1)*t*sd(a$TGiving)/sqrt(numa))

[1] -373.1639  373.1639


In [21]:
ratio=N1/numa
print(ratio)

[1] 13.89771


Question 3: The minimum number of sample size is 17095 with sampling error maximized to 100 on mean total contribution with 95% confidence. With the sampling error decrese to 100 from 373.16 of the original sample, the sample size raise from 1230 to 17095, which is almost 14 times of the original sample size.
